src/modules/todoHelper.js

```diff
@@ -54,6 +53,19 @@ class TodoHelpers {
     return updatedTasks;
   }
 
+  static UpdateClearedTasks(existingTasks) {
+    const updatedTasks = [];
+
+    existingTasks.forEach((task) => {
+      updatedTasks.push({
+        ...task,
+        cleared: task.completed,
+      });
+    });
+
+    return updatedTasks;
+  }
+
   static UpdateTaskStore = (tasks) => {
     localStorage.setItem('tasks', JSON.stringify(tasks));
   };
```

Comment:
- [ ] Great job so far 👍 your site works as expected and the designing is good but there is one small issue to be addressed. After adding, editing or deleting a taks, the updated array of items should be stored in local storage, so the user gets the correct list values after the page reloads but that does not seem to be the case. After I deleted some elements they were still present in the local storage. Kindly address this issue 💯
- Your site:
![Annotation 2022-02-20 062003](https://user-images.githubusercontent.com/85113567/154829721-c919ce56-7522-4a7c-ad8e-de98f89ce1b7.jpg)